deeppavlov/configs/go_bot/gobot_dstc2.json

```diff
@@ -52,6 +52,12 @@
           "primary_keys": ["name"],
           "save_path": "{DOWNLOADS_PATH}/dstc2/resto.sqlite"
         },
+        "nlg_manager": {
```

Comment:
all the gobot configs are updated like

before:
* nlg settings are mixed with the rest of gobot settings
now:
* nlg config is now the gobot config field storing all the NLG-specific settings together.

e.g.:
before
```
{
    ...,
    "template_path": ..., <-- NLG
    "slot_filler": ....   <-- not NLG
    ...
    "template_type": ..., <-- not NLG
}
```
after
```
{
    ...,
    "nlg_manager": {
        "class_name": "gobot_nlg_manager",
        "template_path": "{DATA_PATH}/dstc2-templates.txt",
        "template_type": "DualTemplate",
        "api_call_action": "api_call"
    },
    ...,
    "slot_filler": ....   <-- not NLG
}
```